tkr/webhook/cluster/tkr-resolver/cluster/cluster.go

```diff
@@ -65,6 +65,10 @@ func (cw *Webhook) Handle(ctx context.Context, req admission.Request) admission.
 }
 
 func (cw *Webhook) getClusterClass(ctx context.Context, cluster *clusterv1.Cluster) (*clusterv1.ClusterClass, *admission.Response) {
+	if cluster.Spec.Paused {
```

Comment:
Suggested by @imikushin 

During management cluster deletion, the CLI should replicate TKRs and OSimages (specified by the cluster object label) from MC to the cleanup cluster.